### ISSUE TITLE
[HW1] question in birth day

### ISSUE THREAD

**Issue by wormfry** [swsnu/swppfall2019#14]
> Birth date (Day) : Must be a number of one or two digits.

For birth day requirement, does this mean that 0 is inclusive? and should numbers like 00, 01, 05 be a correct form?

**Comment by hy00nc** [comment]
Yes 0~99 is accepted. For the leading zeros, we won’t test them explicitly, yet it is okay to evaluate them to be correct.
